saleor/checkout/utils.py

```diff
@@ -132,15 +133,23 @@ def add_variant_to_checkout(
         check_quantity=check_quantity,
     )
 
+    if force_new_line:
+        checkout.lines.create(
+            checkout=checkout,
```

Comment:
I suppose you do not have to specify the `checkout` parameter in such case when you are creating the lines with the use of `checkout.lines.create`